extension.json

```diff
@@ -254,7 +254,24 @@
       "schema": {
         "$schema": "http://json-schema.org/draft-04/schema#",
         "type": "object",
-        "additionalProperties": true
+        "additionalProperties": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string"
+              },
+              "authenticatedState": {
+                "type": "string",
+                "enum": ["loggedOut", "authenticated", "ambiguous"]
+              },
+              "primary": {
+                "type": "boolean"
+              }
+            }
```

Comment:
"additionalProperties": false

Reply:
Looking at https://github.com/adobe/reactor-extension-alloy/pull/96/files#diff-5711c4c45d1eca09d94a247045264635R91-R94, it seems that `id` and `authenticatedState` are required. If that's the case, they should be marked required here as well.

Reply:
thank you

Reply:
> Looking at https://github.com/adobe/reactor-extension-alloy/pull/96/files#diff-5711c4c45d1eca09d94a247045264635R91-R94, it seems that `id` and `authenticatedState` are required. If that's the case, they should be marked required here as well.

Actually `authenticatedState` and `primary` are optional with defaults to `ambiguous` and `false`, maybe I should change the Select field to have it default to `ambiguous` and relax validation?

Reply:
Yes ambiguous and false should be the default and relaxing the validation makes sense.